package.json

```diff
@@ -45,54 +45,72 @@
     "redux": ">=2"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.24.1",
-    "babel-eslint": "^7.2.1",
-    "babel-loader": "^7.0.0",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-es2015-classes": "^6.24.1",
-    "babel-plugin-transform-node-env-inline": "^6.8.0",
-    "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-react-jsx": "^6.24.1",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.1",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1",
-    "chai": "^3.5.0",
-    "cross-env": "^4.0.0",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.4.0",
+    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.4.0",
```

Comment:
What's the reason for inlining all of these? I think they're part of `@babel/preset-env`, right?

Reply:
Oh. They're not part of `@babel/preset-env` but they were not used by the config.

I removed most of these packages now and fixed a few things in babel config. Thanks for the observation. :)